app/assets/stylesheets/components/_header.scss

```diff
@@ -0,0 +1,74 @@
+.global-nav {
+  background-color: $c-blue;
+  box-shadow: 2px 2px 4px rgba(black, 0.8);
+  padding: 20px;
+  a {
+  color: white;
+  font-family: $f-logo;
+  margin-left: 4%;
+  text-decoration: none;
+  outline-color: orange;
+  @include fluid-size('font-size', 16px, 27px);
+  }
+
+  a.nav-item {
+    float: right;
+    font-family: $f-body;
+    margin-top: 0.7%;
+    @include breakpoint($small-screen) {
+      margin-top: 0.2%;
+    }
```

Comment:
Some notes on ordering your CSS:
- Mixins first (in case you need to override a mixin with multiple styles)
- 🔤 order all CSS second
- breakpoint code further down, after a new line

Example:
```
 a.nav-item {
    @include fluid-size('font-size', 10px, 21px);
     float: right;
     font-family: $f-body;
     margin-left: 3%;
     margin-right: 4%;
     margin-top: 0.7%;

     @include breakpoint($small-screen) {
       margin-top: 0.2%;
     }
}
```

You can also shorten your `margin` declaration to the short-hand:
`margin: 0.7% 4% 0 3%;`

The short-hand order goes top, right, bottom, left (TRBL or "trouble" helps me remember the order)